fractal/cmd/_task.py

```diff
@@ -143,12 +144,12 @@ async def patch_task(
     meta_file: Optional[str] = None,
 ) -> BaseInterface:
 
-    try:
-        task_id = int(task_id_or_name)
-        if version:
-            logging.warning("Task Version is ignored because Task ID provided")
-    except ValueError:
-        task_name = task_id_or_name
+    if task_id and version:
+        logging.warning(
+            "Too many arguments: cannot provide both `task_id` and `version`."
+        )
+        sys.exit(1)
```

Comment:
Misunderstanding each other here.. ;)

If it's a warning, then we don't exit. We just continue, after ignoring the `version` attribute.